python/cugraph/utilities/utils.py

```diff
@@ -173,6 +176,72 @@ def get_traversed_path_list(df, id):
     return answer
 
 
+def get_traversed_cost(df, source_col, dest_col, value_col):
+    """
+    Take the DataFrame result from a BFS or SSSP function call and sums
+    the given weights along the path to the starting vertex.
+
+    Input Parameters
+    ----------
+    df : cudf.DataFrame
+        The dataframe containing the results of a BFS or SSSP call
+    source_col : cudf.DataFrame
+        This cudf.Series wraps a gdf_column of size E (E: number of edges).
+        The gdf column contains the source index for each edge.
+        Source indices must be an integer type.
+    dest_col : cudf.Series
+        This cudf.Series wraps a gdf_column of size E (E: number of edges).
+        The gdf column contains the destination index for each edge.
+        Destination indices must be an integer type.
+    value_col : cudf.Series
+        This cudf.Series wraps a gdf_column of size E (E: number of edges).
+        The gdf column contains values associated with this edge.
+        Weight can be of any type
```

Comment:
`Weight can be of any type`, can they? Didn't see where integers are handled.

Reply:
Should we add them? I kept it to float for now: https://github.com/rapidsai/cugraph/pull/1376/commits/2c3fc8bdaf9174d71e7842e1f70e571656dc0b7c

Reply:
Well BFS weights should be integers in theory, not sure if it the case in practice

Reply:
Integer support can be done in a separate PR, but we should create an issue for it in this case.